### PR TITLE
(PC-10800) Deeplink generator shorten params description (tech)

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-10800

Amélioration du texte des params description

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXXXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" : `https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a`
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         |                 |                  |                  |

### CODE REVIEW

File: src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts [hunk 1]
```diff
@@ -49,21 +51,21 @@ export const SCREENS_CONFIG: Partial<Record<ScreenNames, ScreenConfig>> = {
 export const MARKETING_CONFIG: Record<string, paramConfig> = {
   utm_campaign: {
     type: 'string',
-    description: `Permet d'identifier le nom de la campagne relative au produit, son slogan ou encore son code promotionnel.`,
+    description: `Identifie le nom de la campagne relative au produit, son slogan ou encore son code promotionnel.`,
```
Comment (BenStirrup) [inline src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts:54]:
```suggestion
    description: `nom de la campagne relative au produit, son slogan ou encore son code promotionnel.`,
```

File: src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts [hunk 2]
```diff
@@ -49,21 +51,21 @@ export const SCREENS_CONFIG: Partial<Record<ScreenNames, ScreenConfig>> = {
...
   },
   utm_source: {
     type: 'string',
-    description: `Permet d'identifier, entre autres, l'annonceur, le site et la publication générant du trafic`,
+    description: `Identifie, entre autres, l'annonceur, le site et la publication générant du trafic`,
```
Comment (BenStirrup) [inline src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts:58]:
```suggestion
    description: `annonceur, site ou publication générant du trafic`,
```

File: src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts [hunk 3]
```diff
@@ -49,21 +51,21 @@ export const SCREENS_CONFIG: Partial<Record<ScreenNames, ScreenConfig>> = {
...
   },
   utm_medium: {
     type: 'string',
-    description: `Permet d'identifier le support publicitaire ou marketing (par exemple : cpc, bannière, newsletter envoyée par e-mail).`,
+    description: `Identifie le support publicitaire ou marketing (par exemple : cpc, bannière, newsletter envoyée par e-mail).`,
```
Comment (BenStirrup) [inline src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts:62]:
```suggestion
    description: `support publicitaire ou marketing (ex : cpc, bannière, newsletter envoyée par e-mail).`,
```

File: src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts [hunk 4]
```diff
@@ -49,21 +51,21 @@ export const SCREENS_CONFIG: Partial<Record<ScreenNames, ScreenConfig>> = {
...
   },
 }
 
 export const FDL_CONFIG: Record<string, paramConfig> = {
   amv: {
     type: 'string',
-    description: `La versionCode de la version minimale de l'application Android qui peut ouvrir le lien (ex: 1016103). Si l'application installée est une version plus ancienne, l'utilisateur est redirigé vers le Play Store pour mettre à niveau l'application.`,
+    description: `La versionCode de la version minimale de l'application Android qui peut ouvrir le lien (ex: ${build}). Si l'application installée est une version plus ancienne, l'utilisateur est redirigé vers le Play Store pour mettre à niveau l'application.`,
```
Comment (BenStirrup) [inline src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts:69]:
```suggestion
    description: `version minimale de l'application Android qui peut ouvrir le lien (ex: amv=${build}). Si l'application installée est une version plus ancienne, l'utilisateur est redirigé vers le Play Store pour mettre à niveau l'application.`,
```

File: src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts [hunk 5]
```diff
@@ -49,21 +51,21 @@ export const SCREENS_CONFIG: Partial<Record<ScreenNames, ScreenConfig>> = {
 export const MARKETING_CONFIG: Record<string, paramConfig> = {
   utm_campaign: {
     type: 'string',
-    description: `Permet d'identifier le nom de la campagne relative au produit, son slogan ou encore son code promotionnel.`,
+    description: `Nom de la campagne relative au produit, son slogan ou encore son code promotionnel.`,
   },
   utm_source: {
     type: 'string',
-    description: `Permet d'identifier, entre autres, l'annonceur, le site et la publication générant du trafic`,
+    description: `Annonceur, le site ou publication générant du trafic`,
```
Comment (BenStirrup) [inline src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts:58]: Si tu retires aux autres noms il faut aussi retirer celui-là
```suggestion
    description: `Annonceur, site ou publication générant du trafic`,
```

File: src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts (hunk 4 above)
Comment (kopax-polyconseil) [reply src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts]: Ok sauf pour `amv=`

File: src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts (hunk 5 above)
Comment (kopax-polyconseil) [reply src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts:58]: Bien vu !

**Review by BenStirrup — APPROVED** [review]

**Pull request merged**